packages/contracts/contracts/Poll.sol

```diff
@@ -186,33 +202,6 @@ contract Poll is Params, Utilities, SnarkCommon, IPoll {
     _;
   }
 
-  /// @notice The initialization function.
-  /// @dev Should be called immediately after Poll creation
-  function init() public virtual {
-    if (isInit) revert PollAlreadyInit();
-    // set to true so it cannot be called again
-    isInit = true;
```

Comment:
What's about maci-platform integration? There is a feature that poll can be created without initialization and after all the data ready (registries, applications) then poll can be initialized.

Reply:
It would not be affected. The maci platform smart contracts already extend the poll factory to change the deploy function, so after this change they would need to add poll.init and implement their own init function inside the poll (or any other function they require).